### PR TITLE
Add LEP for rwx feature

### PR DESCRIPTION
**Pull request by joshimoo**
longhorn/longhorn#1470
longhorn/longhorn#2149

Signed-off-by: Joshua Moody <joshua.moody@rancher.com>

### CODE REVIEW

File: enhancements/20201220-rwx-volume-support.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+### NFS grace period
+- [NFS_lock_recovery_notes](https://linux-nfs.org/wiki/index.php/NFS_lock_recovery_notes)
+- [lower grace period](https://www.suse.com/support/kb/doc/?id=000019374)
+
+### Ceph ganesha
+- [rook crd for ganesha](https://github.com/rook/rook/blob/master/design/ceph/ceph-nfs-ganesha.md)
+- [ganesha driver class library](https://docs.openstack.org/manila/rocky/contributor/ganesha.html)
+
+### NFS ganesha Active Passive Setups
```
Comment (yasker) [inline enhancements/20201220-rwx-volume-support.md:115]: !!! Great find!

File: enhancements/20201220-rwx-volume-support.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+type ShareManagerStatus struct {
+OwnerID  string            `json:"ownerID"`
+State    ShareManagerState `json:"state"`
+Endpoint string            `json:"endpoint"`
+}
+```
+
+
+### Implementation Overview
+- Volume controller is responsible for creation of share manager crd and synconising share status and endpoint of the volume with the share manager resource.
```
Comment (yasker) [inline enhancements/20201220-rwx-volume-support.md:76]: The implementation detail is too brief... We don't even mention how share manager pod mapping to the volume, and how the failure recovery works.

**Review by yasker — CHANGES_REQUESTED** [review]

File: enhancements/20201220-rwx-volume-support.md (hunk 2 above)
Comment (joshimoo) [reply enhancements/20201220-rwx-volume-support.md]: Thanks added more details to the implementation section, please let me know if it is to much detail or additional clarification is required.

Comment (yasker) [reply enhancements/20201220-rwx-volume-support.md]: LGTM now.

**Review by yasker — APPROVED** [review]

**Pull request merged**